packages/lesswrong/components/comments/CommentsItem/CommentsItem.tsx

```diff
@@ -311,6 +349,63 @@ export const CommentsItem = ({
 
   const voteProps = useVote(comment, "Comments", votingSystem);
   const showInlineCancel = replyFormIsOpen && isMinimalist
+  const { create: createDoppelVote, data: newDoppelCommentVote } = useCreate({
+    collectionName: "DoppelCommentVotes",
+    fragmentName: "DoppelCommentVotesFragment",
+  })
+  const {flash} = useMessages()
+  const voteForThisDoppel = (type: "skip" | "vote", doppelCommentChoiceId?: string) => {
+    if (!currentUser) return
+    void createDoppelVote({
+      data: {
+        userId: currentUser._id,
+        commentId: comment._id,
+        type,
+        doppelCommentChoiceId,
+      },
+    })
+    type === "vote" && flash(`You were ${doppelCommentChoiceId === null ? "correct" : "incorrect"}`)
+  }
+  const doppelCommentVote = (className: string) => (comment: CommentsList) => {
+    return <>
+      <div className={className}>
+        <LWTooltip title="Do you think the current comment was really written by the user, rather than an LLM?">
+          <Button className={classes.doppelCommentsButton} onClick={_ => voteForThisDoppel("vote", potentialCommentBodies[activePotentialComment].doppelId ?? undefined)}>
+            Pick current
+          </Button>
+        </LWTooltip>
+        <LWTooltip title="Reveal the true comment without voting">
+          <Button className={classes.doppelCommentsButton} onClick={_ => voteForThisDoppel("skip")}>
+            Skip
+          </Button>
+        </LWTooltip>
+        <LWTooltip title="Don't see these Turing tests on any comments">
+          <Button className={classNames(classes.doppelCommentsButton, classes.doppelCommentsOptOutButton)} onClick={_ => setAdminTriedToOptOut(comment._id)}>
+            Opt out of this feature
+          </Button>
+        </LWTooltip>
+      </div>
+      {adminTriedToOptOut === comment._id && <div className={classes.adminTriedToOptOutMessage}>Admins can't opt out of this "feature"</div>}
+    </>
+  }
+
+  const shouldWeDoppel = isLWorAF // forum gate
+    && currentUser?.isAdmin // admin gate
+    && comment.doppelComments.length >= 2 // are there doppel comments to show?
+    && !(comment.ownDoppelCommentVote || newDoppelCommentVote) // has user already voted?
+
+  const potentialCommentsData = shouldWeDoppel
```

Comment:
Are we using `potentialCommentsData` after this except as an existince check on line 403?

Reply:
ya, used by `CommentsBody`